c_src/Makefile

```diff
@@ -39,7 +39,7 @@ endif
 CFLAGS += -fPIC -I $(ERTS_INCLUDE_DIR) -I $(ERL_INTERFACE_INCLUDE_DIR)
 CXXFLAGS += -fPIC -I $(ERTS_INCLUDE_DIR) -I $(ERL_INTERFACE_INCLUDE_DIR)
 
-LDLIBS += -L $(ERL_INTERFACE_LIB_DIR) -lerl_interface -lei -ldbus-1
```

Comment:
@JayKickliter has asked that we remove ei too, since NIFs don't use it and that might save us some trouble down the road.

Reply:
No objections.

Reply:
done